app/app.go

```diff
@@ -300,17 +282,16 @@ func fetchFullExit(ctx context.Context, eth2Cl eth2wrap.Client, oAPI obolapi.Cli
 	return true
 }
 
-func postPartialExit(ctx context.Context, oAPI obolapi.Client, mutationHash []byte, shareIndex uint64, identityKey *k1.PrivateKey, exitBlobs ...obolapi.ExitBlob) bool {
+// postPartialExit posts exitBlobs to Obol API with a 10 seconds HTTP request deadline.
```

Comment:
We should avoid adding specific values in godoc like 10 seconds, as it will become outdated if actual timeout value changes in the code. Can have a constant variable.

Reply:
Fixed!